src/services/actions/CliActions.ts

```diff
@@ -432,11 +423,11 @@ export class CliActions {
     });
   }
 
-  /**
-     * Adds a Tenant App Catalog.
-     * The URL is fixed to "https://domain.sharepoint.com/sites/appcatalog".
-     * Prompts the user for the owner and timeZone.
-     */
+/**
+   * Adds a Tenant App Catalog.
+   * The URL is fixed to "https://domain.sharepoint.com/sites/appcatalog".
+   * Prompts the user for the owner and timeZone.
+   */
   public static async addTenantAppCatalog() {
```

Comment:
lets align the spacing so the comment will be on the same level as the public method
```suggestion
  /**
     * Adds a Tenant App Catalog.
     * The URL is fixed to "https://domain.sharepoint.com/sites/appcatalog".
     * Prompts the user for the owner and timeZone.
     */
  public static async addTenantAppCatalog() {
```